### PR TITLE
docs: sp docs add flowchart

### PR DESCRIPTION
**Pull request by sysvm**
### Description

SP docs add flowchart.

### Rationale

N/A

### Example

N/A

### Changes

Notable changes: 
* N/A

### CODE REVIEW

File: docs/asset/07-challenge.jpg
Comment (will-2012) [inline docs/asset/07-challenge.jpg:1]: `lenfgth` typo.

**Review by will-2012 — APPROVED** [review]

File: docs/asset/08-gc_object.jpg
Comment (will-2012) [inline docs/asset/08-gc_object.jpg:1]: Add `4. Delete integrity meta SPDB`.

File: docs/asset/07-challenge.jpg
Comment (sysvm) [reply docs/asset/07-challenge.jpg:1]: fixed

File: docs/asset/08-gc_object.jpg
Comment (sysvm) [reply docs/asset/08-gc_object.jpg:1]: added

**Review by joeylichang — APPROVED** [review]

**Pull request merged**